python/lsst/ts/m2gui/utils.py

```diff
@@ -52,3 +64,23 @@ def set_button(name, callback, *args, is_checkable=False):
     button.clicked.connect(partial(callback, *args))
 
     return button
+
+
+def get_rgb_from_hex(code):
```

Comment:
Do you really need that? QColor takes as possible argument # string:

```python
In [3]: from PySide2.QtGui import QColor
In [4]: QColor('#0f0f0f')
Out[4]: PySide2.QtGui.QColor.fromRgbF(0.058824, 0.058824, 0.058824, 1.000000)
```

It is documented in C++ doc for QColor, not PySide2:

void QColor::setNamedColor(const [QString](https://doc.qt.io/qt-5/qstring.html)[](https://doc.qt.io/qt-5/qcolor.html#setNamedColor) &name)

Sets the RGB value of this [QColor](https://doc.qt.io/qt-5/qcolor.html) to name, which may be in one of these formats:

#RGB (each of R, G, and B is a single hex digit)
#RRGGBB
#AARRGGBB (Since 5.2)
#RRRGGGBBB
#RRRRGGGGBBBB
A name from the list of colors defined in the list of [SVG color keyword names](https://www.w3.org/TR/SVG11/types.html#ColorKeywords) provided by the World Wide Web Consortium; for example, "steelblue" or "gainsboro". These color names work on all platforms. Note that these color names are not the same as defined by the [Qt::GlobalColor](https://doc.qt.io/qt-5/qt.html#GlobalColor-enum) enums, e.g. "green" and [Qt::green](https://doc.qt.io/qt-5/qt.html#GlobalColor-enum) does not refer to the same color.
transparent - representing the absence of a color.
The color is invalid if name cannot be parsed.

from https://doc.qt.io/qt-5/qcolor.html#QColor-5 (note that's case with PySide2 Qt doc, which is lacking official Qt in some areas; but PySide2 basically just encapulates C++ calls, so usually what you can use in C++, you can use in PySide2 (with some exceptions..)).

Reply:
I added the function based on a tutorial to use the `Color` in `Qt`. I think the reason to have this function is improve the readability (not use the hexadecimal word directly, but use a dictionary of colors with the hexadecimal words). It just calls the `QColor.fromRgb()`, which I felt the same as `QColor.fromRgbF()` with the difference of data type only. Please let me keep this function for the moment to see I really need this function or not. Thanks for the suggestion!